packages/@integration/test/integration.spec.ts

```diff
@@ -71,6 +71,15 @@ describe('API integration', () => {
         expect(body).toHaveLength(3);
       }));
 
+  test(`GET returns 200 with Content-Type=application/json: /api/v1/user?email=test@test.com`, async () =>
+    request(server)
+      .get('/api/v1/user?email=test@test.com')
+      .set('Authorization', 'Bearer FAKE')
```

Comment:
This is not a valid url. `@` character should be url-encoded. Still, it doesn't change the outcome of the test (and the test is still valid)

Reply:
Browsers don’t complain about it